compensated-ruby/lib/compensated/event_parser.rb

```diff
@@ -1,17 +1,23 @@
 module Compensated
   class EventParser
     def parse(request)
-      normalize(request.data)
+      transform(request.data)
     end
 
     def parses?(request)
       raise NotImplementedError("Implement in Child")
     end
 
-    def normalize(data)
+    def transform(data)
       raise NotImplementedError("Implement in Child")
     end
 
+    # <b>DEPRECATED:</b> Please use <tt>transform</tt> instead.
+    def normalize(data)
+      warn '[DEPRECATION] `normalize` is deprecated.  Please use `transform` instead.'
+      transform(data)
```

Comment:
Blocking: I think `normalize` wants to call `transform(extract(data))` so that `transform` doesn't need to worry about `extract`